doc/mpich/TUNING_PARAMETERS

```diff
@@ -43,12 +43,26 @@ VCI lock and a `lockless` mode where MPI does not grab a lock and relies on the
 network provider to provide fine grain synchronization (MPICH uses the
 FI_THREAD_SAFE mode from OFI to support the `lockless` mode).
 
+A threading synchronization strategy can be selected at the configuration time
+as:
+  --enable-thread-cs=global to select coarse-grained locking, and
+  --enable-thread-cs=per-vci to select fine-grained locking.
```

Comment:
Can you break the line so the command snippet is on its own line?

Reply:
I was debating with myself about it. Sure, let me do that.

Reply:
Done